tokio/src/runtime/queue.rs

```diff
@@ -79,6 +52,13 @@ const LOCAL_QUEUE_CAPACITY: usize = 4;
 
 const MASK: usize = LOCAL_QUEUE_CAPACITY - 1;
 
+fn make_fixed_size<T>(buffer: Box<[T]>) -> Box<[T; LOCAL_QUEUE_CAPACITY]> {
+    assert_eq!(buffer.len(), LOCAL_QUEUE_CAPACITY);
+
+    // SAFETY: We check that the length is correct.
```

Comment:
nit: all the other safety comments in this file are all lowercase...we should probably make this one consistent
```suggestion
    // safety: We check that the length is correct.
```

(i would also be fine with changing all the other comments to write `SAFETY` rather than `safety`, but i'd prefer them to all be the same :) )